_posts/en/meetings/2018-07-12-meeting.md

```diff
@@ -0,0 +1,231 @@
+---
+title: IRC meeting summary for 2018-07-12
+lang: en
+permalink: /en/meetings/2018/07/12/
+name: 2018-07-12-meeting
+layout: page
+type: meetings
+version: 1
+---
+{% include toc.html %}
+{% include references.md %}
+
+- View this week's log on [BotBot.me][bbm log] or [MeetBot][mb log]
+- [Meeting minutes by MeetBot][mb minutes]
+
+---
+
+Topics discussed during this weekly meeting included an announcement of
+a poll to help find widely-acceptable times for future weekly meetings,
+whether or not to move back the planned feature freeze date for Bitcoin
+Core 0.17, whether or not to produce a 0.16.2 maintenance release (and
+when to do it), status of upgrading the deterministic build environment
+to a new version of Ubuntu, and the status of a PR related to what coins
+(inputs) the wallet automatically chooses to spend.
+
+The usual first topic of *high priority pull requests for review* was
+skipped.  Wladimir var der Laan said, "I've intentionally skipped high
+priority for review this time.  [It] seems pretty clear [that we] just
+need to get the features for 0.17 in ASAP."
+
+## Meeting time poll
+
+**Background:** as mentioned [last meeting][], the project is trying to
+determine the best times to hold weekly meetings.
+
+**Discussion ([log][log meeting time]):** Cory Fields said, "Remember to
+vote on meeting time.  If you didn't get a mail about it, now's the time
+to tell me! [The poll closes] this time next week."
+
+**Conclusion:** if you're a regular contributor, check your email for a
+poll about it or contact Fields.
+
+## Move feature freeze date?
+
+**Background:** Bitcoin Core attempts to release a new major version
+every six months.  The first step of the release process is freezing the
+current feature set so that the focus can shift to maturing, testing,
+and documenting the upcoming release.
+
+**Discussion ([log][log feature freeze]):** Wladimir van der Laan
+introduced the topic, "So the current feature freeze is July 16th, which
+is in a few days [...] my question is, should we delay it, [or] are
+there important things that [we] would otherwise miss but [which] are
+*almost* ready?"
+
+Several meeting participants listed things they'd like to get in that
+they're worried won't be complete before the freeze:
+
+- [New scantxout RPC][#12196], mentioned by Jonas Schnelli
+- [New `disableprivatekey` setting][#9662] for watching-only wallets,
```

Comment:
Same feedback as the line above: remove backticks to avoid strange-looking link.

Reply:
I fixed, squashed and force-pushed this to not delay the merge. Hope that's ok @harding!

Reply:
(I assume so since you'd already done the same fix to the link above)

Reply:
LGTM, thanks!  I'll wait another couple hours until around 16:00 EDT in case anyone else wants to comment, then merge.